packages/replay-next/pages/variables.css

```diff
@@ -1038,6 +1039,8 @@
   --testsuites-steps-bgcolor: #fff;
   --testsuites-steps-toggle-bgcolor: #fff;
   --testsuites-success-color: #058b00;
+  --testsuites-skipped-color: var(--theme-base-70);
```

Comment:
This variable was missing. For some reason it never got added for the light theme (cc @ryanjduffy wrt #8839)